spring-boot-project/spring-boot-docs/pom.xml

```diff
@@ -960,6 +969,31 @@
 	<profiles>
 		<profile>
 			<id>full</id>
+			<dependencies>
+				<dependency>
+					<groupId>org.springframework.doc-resources</groupId>
```

Comment:
This is quite an unconventional group id due to the `-` in it. I would have raised an issue for it in https://github.com/spring-projects/spring-doc-resources but issues appear to be disabled and there's no link to an alternative issue tracker in the README. Could it be changed pleased?

Reply:
Should it be org.springframework.doc.resources?

Reply:
I think `docresources` would be more conventional. It would also match other projects like REST Docs (restdocs) and Web Flow (webflow) too.

There’s also a case to be made for it to be `io.spring` rather than `org.springframework`. The latter is typically reserved for projects that are used by the community when developing an application, have a programmatic API, etc.

Reply:
I'll change it to io.spring.docresources once Brian has finished his review. We have four projects (and growing) that use this set of resources now, so I'll have some changes to make.